src/scripts/constants.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class _constants:
+
+    _DEFAULT_TEST_COUNT = 100 # The Default value of --test-count option
```

Comment:
The underscore prefix is OK for me for the class name, but I'd remove them from the class members.

Reply:
`Default` -> `default`